lib/OpenQA/CacheService.pm

```diff
@@ -32,7 +32,9 @@ sub startup {
     # Worker settings
     my $global_settings = OpenQA::Worker::Settings->new->global_settings;
     my $location        = $ENV{OPENQA_CACHE_DIR} || $global_settings->{CACHEDIRECTORY};
-    my $limit           = $global_settings->{CACHELIMIT};
+    die "Cache directory unspecified. Set environment variable 'OPENQA_CACHE_DIR' or config variable 'CACHEDIRECTORY'."
```

Comment:
I'd prefer a "\n" at the end of the message so it looks less like an internal error and more like a user-facing error message.

Reply:
Hm, I'm torn. On one hand, yes, looks less like a "usual internal perl warning not of my concern" but the location in file might still be helpful. But the message should be clear enough and is limited to the cache service. I guess I can fulfill your wish :)